lessons/basics/strings.md

```diff
@@ -18,31 +18,32 @@ Elixir strings are nothing but a sequence of bytes. Let's look at an example:
 ```elixir
 iex> string = <<104,101,108,108,111>>
 "hello"
+iex> string <> <<0>>
+<<104, 101, 108, 108, 111, 0>>
 ```
 
+By concatenating the string with the byte 0, IEx displays the string as a binary because it is not a valid string anymore. This trick can help us view the underlying bytes of any string.
+
 >NOTE: Using << >> syntax we are saying to the compiler that the elements inside those symbols are bytes.
 
 ## Char Lists
 
 Internally, Elixir strings are represented with a sequence of bytes rather than an array of characters. Elixir also has a char list type (character list). Elixir strings are enclosed with double quotes, while char lists are enclosed with single quotes.
 
-What's the difference? Each value from a char list is the ASCII value from the character. Let's dig in:
+What's the difference? Each value in a char list is the unicode code point of a character whereas in a binary, the codepoints are encoded as UTF-8. Let's dig in:
```

Comment:
We are no longer referring to "char lists" but to "charlists". Actually we have replaced all functions that used the old naming `char_list`.

Please also change the title of the section.